### DISCUSSION TITLE
It is posible to open office files online like "view online" feature of ms onedrive?

### DISCUSSION THREAD

**Discussion by matiaschn** [abraunegg/onedrive#2988] [Q&A] [answered]
I would like to know if there is any way to open the word and excel files that are stored in the synchronized directory in the office 365 online apps. Something like "view online" feature of MS onedrive. 

Thanks

**Comment by abraunegg** [comment, marked as answer]
That would be a function of your GUI - gnome / kde / other to provide the menu item when you right click on a supported file type to perform that action

There is zero way for this client to do this

Your best course of action is to raise a query or feature request with the gnome / kde / other developers if this does not exist today.
